parts/xmem_nozzle.go

```diff
@@ -1649,13 +1644,20 @@ func (xmem *XmemNozzle) batchGetMeta(bigDoc_map base.McRequestMap) (map[string]N
 					sourceDoc := crMeta.NewSourceDocument(wrappedReq, xmem.sourceBucketId)
 					sourceMeta, err := sourceDoc.GetMetadata()
 					if err != nil {
-						xmem.Logger().Warnf("%v batchGetMeta: Error decoding source document metadata for doc %v%q%v. err=%v. Skip conflict resolution and send the doc", xmem.Id(), base.UdTagBegin, key, base.UdTagEnd, err)
+						xmem.Logger().Warnf("%v batchGetMeta: doc %v%q%v failed source side conflict resolution, but had error decoding source document metadata for logging. err=%v.",
```

Comment:
For next PR: Add a counter internally in XMEM for this - and add other counters for below as necessary
No need to export to prometheus for now, but should at least be logged in xmem periodic summary

Reply:
Noted